jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/SyncServerlessAppAction.kt

```diff
@@ -75,8 +75,8 @@ class SyncServerlessAppAction(private val codeOnly: Boolean = false) : AnAction(
             }
 
             val execVersion = SemVer.parseFromText(samExecutable.version) ?: error("SAM CLI version could not detected")
-            val minVersion = SemVer("1.53.0", 1, 53, 0)
-            val minVersionForUseContainer = SemVer("1.57.0", 1, 57, 0)
+            val minVersion = SemVer("1.78.0", 1, 78, 0)
```

Comment:
maybe we should use the breaking change changelog entry

Reply:
Added changelog